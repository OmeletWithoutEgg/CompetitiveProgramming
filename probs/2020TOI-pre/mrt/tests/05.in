57
37 5 6 19 35 35 5
7 4 6 5 5 6 6
37 7 34 13 18 12 25
37 13 11 21 6 18 3
5 3 2 2 4 1 4
2 1 1 0 1 1 1
53 5 49 29 16 37 31
47 12 17 37 42 15 0
11 9 7 9 5 2 0
41 11 12 24 18 16 15
3 2 0 1 0 2 0
53 2 6 18 16 23 20
37 16 9 31 17 6 5
29 18 17 7 15 17 4
19 14 0 5 9 8 15
2 0 1 0 1 1 0
47 30 14 40 42 39 25
67 63 15 31 1 26 0
37 4 27 35 14 32 17
3 2 1 2 2 1 2
11 6 2 3 5 9 9
61 55 7 31 37 18 27
13 2 7 3 12 1 6
61 5 58 55 19 22 43
41 1 14 4 0 20 11
29 16 3 13 1 21 28
43 7 18 20 17 26 19
29 16 13 6 21 1 28
41 25 19 1 39 16 28
23 12 15 21 12 2 1
59 56 1 12 10 39 12
2 0 1 1 0 0 1
13 1 8 3 5 6 10
5 4 4 1 2 2 0
61 41 58 9 27 17 42
7 2 3 2 1 6 0
5 2 1 0 4 3 3
17 0 9 14 4 0 14
47 16 31 2 27 28 23
41 31 29 17 37 1 15
53 36 52 7 45 23 50
7 5 3 2 0 6 2
19 6 1 6 14 7 5
53 14 23 49 50 40 33
47 45 8 40 33 19 25
67 22 49 11 58 0 7
31 2 2 3 29 17 19
2 1 1 0 1 1 1
53 48 1 14 5 31 13
3 2 1 2 1 0 2
17 3 0 3 11 5 15
53 16 35 1 23 48 28
67 65 37 54 12 57 14
59 10 44 47 5 12 2
23 22 18 21 9 16 21
53 48 15 10 45 9 13
5 1 1 2 4 3 4
