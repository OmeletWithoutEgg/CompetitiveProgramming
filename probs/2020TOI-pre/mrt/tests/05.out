16 36
4 4
24 15
36 21
2 4
1 1
46 28
27 40
5 10
21 22
0 0
19 38
1 10
21 17
10 0
0 1
22 18
2 22
8 4
0 2
8 1
31 37
5 5
42 34
25 17
15 15
21 3
17 23
28 11
9 22
11 58
0 1
1 6
1 2
49 41
3 6
1 2
16 5
44 35
14 38
32 23
1 3
11 7
33 8
2 23
26 24
21 25
1 1
46 45
2 1
13 14
15 29
16 9
12 2
12 3
51 32
0 4
